aws_lambda_builders/actions.py

```diff
@@ -156,7 +156,11 @@ def __init__(self, source: Union[str, os.PathLike], dest: Union[str, os.PathLike
         self._dest = dest
 
     def execute(self):
+        source_path = Path(self._source)
         destination_path = Path(self._dest)
+        if not source_path.exists():
+            # Source path doesn't exist, nothing to symlink
+            return
```

Comment:
Shouldn't this be the case when this action tries to create a symlink? This will try to create a symlink from `/foo/bar` to `/foo/baz`. In this case, `/foo/bar` (`source_path`) shouldn't exist but `/foo/baz` (`destination_path`) should already exist.

Reply:
In this case the symlink would create `/foo/baz` and link it to `/foo/bar` even though `/foo/bar` doesn't exist. The symlink itself will be created successfully even though it's not linking to anything.